tutorials/spherex/spherex_intro.md

```diff
@@ -112,10 +112,10 @@ len(results)
 The query results provide a lot of metadata about each spectral image. These columns have standard names as defined by the IVOA. Let's list them:
 
 ```{code-cell} ipython3
-list(results.fieldnames)
+results.colnames
 ```
 
-The 'access_url' column is particularly important because it tells you how to access the data. Let's look at the 'access_url' value for the first row:
+The `'access_url'` column is particularly important because it tells you how to access the data. Let's look at the `'access_url'` value for the first row:
 
 ```{code-cell} ipython3
 results['access_url'][0]
```

Comment:
The PR is draft as this line currently returns an incomplete HTTPS url rather than a datalink.

Reply:
This appears to have been fixed, at least in the sense that it is now returning a full url instead of a relative path.

```python
>>> results = Irsa.query_sia(pos=(coord, search_radius), collection='spherex_qr')
>>> results['access_url'][0]
'https://irsa.ipac.caltech.edu/ibe/data/spherex/qr/level2/2025W20_2D/l2b-v12-2025-176/6/level2_2025W20_2D_0444_1D6_spx_l2b-v12-2025-176.fits'
```

Reply:
Thanks, that's great news!